stack/stack_gfsv16.2.yaml

```diff
@@ -0,0 +1,359 @@
+cmake:
+  build: NO
+  bootstrap: NO
+  version: 3.20.1
```

Comment:
We have this here as a placeholder, and it's set to not build because we use the system CMake. I can put in a request to Orion/Hera admins to have CMake 3.20.2 installed if you want to be consistent.

Reply:
If you wouldn't mind asking the Orion/Hera admins to install cmake 3.20.2 that would be great, thanks a bunch!